metricbeat/modules.d/linux.yml.disabled

```diff
@@ -0,0 +1,18 @@
+# Module: linux
+# Docs: https://www.elastic.co/guide/en/beats/metricbeat/master/metricbeat-module-linux.html
+
+- module: linux
+  period: 10s
+  metricsets:
+    - "pageinfo"
+    # - ksm
+    # - conntrack
+    # - perf
+  enabled: true
+  #hostfs: /hostfs
+  #perf.sample_period: 3s
```

Comment:
Should this be commented out if it's required?

Reply:
It's only required for the `perf` metricset?

Reply:
Ah yes, fair point.